pkg/auto/lights/auto.go

```diff
@@ -181,25 +185,32 @@ func (b *BrightnessAutomation) processStateChanges(ctx context.Context, readStat
 			}
 
 			// if the context remains live, schedule another update soon
-			// TODO: make this delay configurable, or add backoff etc.
-			after := 5 * time.Second
+			b.retries++
+			after := time.Duration(b.retries) * b.backoffMultiplier
```

Comment:
We already use a [backoff package](https://pkg.go.dev/github.com/cenkalti/backoff/v4) in sc-bos, which can help with things like randomised exponential backoff.
You could consider using it here, although a manual implementation is fine too.

Other things I can see:
 - If `b.retries == 0`, then `ttl` will be set to 0, and the retry will be immediate. I think there should be at least a short delay.

Reply:
> My understanding of exponential backoff is it tends to be required in heavy or slow operations where receiving 2 requests simultaneously may be unnecessary or have side-effects. Otherwise, constant or linear backoff is fine. Here, the error is in setting the brightness level of a device, which I imagine is a small network request that doesn't necessarily take a while. It's just that the request may fail sometimes. So linear backoff is sufficient? Again, my understanding of devices and how they work is fairly limited.

Some of the uses of this automation are with lights controlled over the DALI protocol, which is really slow (~100ms / operation). With one automation, that's no problem. But with dozens of automations controlling lights on the same DALI bus, if there's no randomisation then the retries for all will tend to happen at the same time, which would cause a lot of bus congestion.